### PR TITLE
Fix SemVer and RagDoll features

### PR DESCRIPTION
**Pull request by warden161**
SemVer says that [beta versions should be 0.1.0](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)

Also added some stuff for ragdoll lol

### CODE REVIEW

File: CursedMod/Features/Wrappers/Player/RagDolls/CursedRagDoll.cs [hunk 1]
```diff
@@ -1,4 +1,10 @@
-﻿using Mirror;
+﻿using CursedMod.Features.Wrappers.Server;
+using InventorySystem.Items.Pickups;
+using Mirror;
+using PlayerRoles;
+using PlayerStatsSystem;
+using System;
+using System.CodeDom;
```
Comment (xNexusACS) [inline CursedMod/Features/Wrappers/Player/RagDolls/CursedRagDoll.cs]: System.CodeDom? are you using the reference for something?

Comment (warden161) [reply CursedMod/Features/Wrappers/Player/RagDolls/CursedRagDoll.cs]: I don't know where but I am
EDIT: visual studio did this for some reason, I'm not using it.

Comment (xNexusACS) [reply CursedMod/Features/Wrappers/Player/RagDolls/CursedRagDoll.cs]: well, you can always delete the reference if you're not using it to make it more clean and stetic

**Review by xNexusACS — APPROVED** [review]

**Comment by warden161** [comment]
sorry about fix things commit i was working on something and forgot to switch branches

**Comment by Jesus-QC** [comment]
I am sorry but this wont be accepted. The only change I would accept is the semver fix. If you make it feel free to reopen

**Pull request closed without merging**